examples/language/gpt/requirements.txt

```diff
@@ -1,3 +1,3 @@
-colossalai >= 0.1.12
-torch >= 1.8.1
-transformers >= 4.231
+psutil
+transformers >= 4.23
+ninja
```

Comment:
`psutil` and `ninja` will be installed when `colossalai` is installed, so there is no need to install them here.